service/controller/v11/key/key.go

```diff
@@ -89,6 +89,14 @@ func BucketObjectName(templateVersion string, prefix string) string {
 	return fmt.Sprintf("cloudconfig/%s/%s", templateVersion, prefix)
 }
 
+func CredentialName(customObject v1alpha1.AWSConfig) string {
+	return customObject.Spec.AWS.Credential.Name
```

Comment:
This is now `customObject.Spec.AWS.CredentialSecret.Name`. Pleas update `apiextensions` dependency before merging.

Reply:
done.

also updating aws-resource-lab